deployment/README.md

```diff
@@ -1 +1,47 @@
 # Deployment
+
+## Requirements
+
+* Helm
+* [helm-diff plugin](https://github.com/databus23/helm-diff)
+* helmfile
```

Comment:
I can't find any official site for this, maybe https://github.com/roboll/helmfile ? If so, better add it as a link.

Reply:
helm plugin: `helm plugin install https://github.com/databus23/helm-diff`
helmfile: 
Download the latest version: `https://github.com/roboll/helmfile/releases/`
for example: 
`wget https://github.com/roboll/helmfile/releases/download/v0.134.0/helmfile_linux_amd64 -O helmfile_linux_amd64`
`chmod +x helmfile_linux_amd64`
`sudo mv helmfile_linux_amd64 /usr/local/bin/helmfile`

Reply:
Can you update the document to add these instructions?